src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,50 @@
-export const Tabs = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { Tab } from '../../Types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  onTabSelected: (tab: Tab) => void;
+  selectedTab: Tab;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  onTabSelected,
+  selectedTab,
+
+}) => {
+  const selectTab = (tab: Tab) => {
+    if (tab.id !== selectedTab.id) {
+      onTabSelected(tab);
+    }
+  };
+
```

Comment:
Here you must find `selectedTab` by id from props and if you don't have it use `tabs[0]` as selected tab

Reply:
fixed 👌